### PR TITLE
Helm install via yum

### PR DESCRIPTION
**Pull request by karthicgit**
Kubernetes version changes in doc .Changed helm.tf to kubernetestools.tf as discussed.please review.

### CODE REVIEW

File: docs/terraformoptions.adoc [hunk 1]
```diff
@@ -452,8 +452,8 @@ admission_controller_options = {
 
 |`kubernetes_version`
 |The version of Kubernetes to provision. This is based on the available versions in OKE. By default, the latest version is selected. The use of 'LATEST' is no longer permitted in order to facilitate upgrades.
-|v1.14.8,v1.15.7, v1.16.8
-|v1.16.8
+|v1.15.7, v1.16.8 ,v1.17.9
```
Comment (hyder) [inline docs/terraformoptions.adoc:455]: Create a new issue, mention this and link it to this PR.

File: modules/oke/kubernetestools.tf [hunk 2]
```diff
@@ -4,10 +4,6 @@
 data "template_file" "helm_enabled" {
```
Comment (hyder) [inline modules/oke/kubernetestools.tf:4]: Rename to install_helm

File: modules/oke/kubernetestools.tf [hunk 3]
```diff
@@ -4,10 +4,6 @@
...
   template = file("${path.module}/scripts/install_helm.template.sh")
 
-  vars = {
-    helm_version       = var.helm.helm_version
-  }
-
   count = var.oke_operator.operator_enabled == true && var.helm.helm_enabled == true ? 1 : 0
```
Comment (hyder) [inline modules/oke/kubernetestools.tf:7]: No need to check if helm is enabled as discussed. So check only if operator is enabled.

File: modules/oke/kubernetestools.tf [hunk 4]
```diff
@@ -24,7 +20,7 @@ resource null_resource "install_helm_operator" {
     bastion_private_key = file(var.oke_ssh_keys.ssh_private_key_path)
   }
 
-  depends_on = [null_resource.install_kubectl_operator, null_resource.write_kubeconfig_on_operator]
+  depends_on = [null_resource.install_kubectl_operator]
 
   provisioner "file" {
```
Comment (hyder) [inline modules/oke/kubernetestools.tf:25]: Move this provisioner into the install kubectl_operator.

File: terraform.tfvars.example [hunk 5]
```diff
@@ -179,8 +179,6 @@ username = ""
 # helm
 helm_enabled = false
```
Comment (hyder) [inline terraform.tfvars.example:180]: Remove from here, all the various variables.tf as well as the docs.

File: variables.tf [hunk 6]
```diff
@@ -433,12 +433,6 @@ variable "helm_enabled" {
   type        = bool
 }
 
-variable "helm_version" {
```
Comment (hyder) [inline variables.tf:436]: Excellent. As long as version 3.x is installed from the olcne repo, we can install it.

**Comment by hyder** [comment]
Also, rename the install kubectl null_resource to install kubernetes tools or something similar. So, we'll use only 1 null_resource and multiple file provisioners and run each script separately.

File: modules/oke/kubernetestools.tf (hunk 2 above)
Comment (karthicgit) [reply modules/oke/kubernetestools.tf]: done.

File: modules/oke/kubernetestools.tf (hunk 3 above)
Comment (karthicgit) [reply modules/oke/kubernetestools.tf]: helm variables were removed

**Comment by karthicgit** [comment]
@hyder  base version changed to 1.3.3. helm variables has been removed and doc changes done.

**Review by hyder — COMMENTED** [review]
Will make some additional changes.

**Review by hyder — APPROVED** [review]
making some cosmetic changes

**Pull request merged**